packages/ui/components/primitives/ResponsivePayloadImage/index.tsx

```diff
@@ -165,7 +174,8 @@ function ResponsivePayloadImage({
         }
         sizes={fill ? fillSizes : '100vw'}
         alt={alt ?? ''}
-        style={{ width: fill ? '100%' : 'initial' }}
```

Comment:
Moving these out of inline styles so they can be overwritten without extreme css specificity